react/components/index.js

```diff
@@ -59,28 +61,34 @@ const LocaleSwitcher = ({ data }) => {
     return null
   }
 
+  const containerClasses = classnames(handles.container, 'w3 flex items-center justify-end ml2 mr3 relative')
+  const buttonClasses = classnames(handles.button, 'link pa0 bg-transparent bn flex items-center pointer mr3 c-on-base')
+  const buttonTextClasses = classnames(handles.buttonText, 'pl2 t-action--small order-1')
+  const listClasses = classnames(handles.list, 'absolute z-5 list top-1 w3 ph0 mh0 mv4 bg-base')
+  const listElementClasses = classnames(handles.listelement, 't-action--small pointer f5 pa3 hover-bg-muted-5 tc')
```

Comment:
I would not use `classnames` here (or ever, I'm biased against it because I think it's unnecessary hahaha). Why not just use a template string? Your app would have one less dependency :)